.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,7 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.154.0/containers/ruby
 {
     "name": "Ruby",
-    "image": "ghcr.io/truecharts/devcontainer:v2.3.0@sha256:a163ad40427cd75a31a0365afb832f97e512d420429ed091b90b18a034526264"
+    "image": "ghcr.io/truecharts/devcontainer:v2.3.0@sha256:a163ad40427cd75a31a0365afb832f97e512d420429ed091b90b18a034526264",
```

Comment:
I prefer not adding unrelated changes

Reply:
Sure. I just thought since there is a obvious syntax error I might fix it while I'm there.